pkg/gui/gui.go

```diff
@@ -429,6 +471,13 @@ func (gui *Gui) quit(g *gocui.Gui, v *gocui.View) error {
 		return gui.createUpdateQuitConfirmation(g, v)
 	}
 	if gui.Config.GetUserConfig().GetBool("confirmOnQuit") {
+		// clear command status buffer
```

Comment:
I'm not sure why we're clearing this buffer if we're about to quit anyway

Reply:
It is seems unnecessary to do this XD